crates/eigenda/src/eigenda_preimage.rs

```diff
@@ -94,6 +78,50 @@ where
     }
 }
 
+fn recency_check(
+    l1_inclusion_bn: u64,
+    rbn: u64,
+    recency_window: u64,
+) -> Result<(), HokuleaStatelessError> {
+    // unlike the go implementation, it skips check when l1_inclusion_bn is 0. Go implementation needs this because
+    // this number is passed from the op-node to the golang proxy. To handle older op-node that does not pass the
+    // value, l1_inclusion_bn = 0 is treated as the default. For hokulea, l1_inclusion_bn is derived directlu, hence
+    // there is no chance, l1_inclusion_bn can be zero.
+    // https://github.com/Layr-Labs/eigenda/blob/cd5c80ad88e966844a4e77795bda07c73dd772ba/api/proxy/store/generated_key/v2/eigenda.go#L319
+    if recency_window == 0 {
+        warn!(
+            "recency check is disabled in the eigenda blob derivation. \
+                It is vulnerable to malicious or misbehaving batcher that \
+                submits DA certificate whose blob has been pruned by the \
+                DA network"
+        );
+        return Ok(());
+    }
+
+    // operator is not allowed to use the genesis block, because there is no operators stake
+    if rbn == 0 {
```

Comment:
thinking about this more I guess its wrong? Since we could technically start a devnet chain with operator stake setup... And then our disperser's policy of using `cur_block - 75` could def lead to using RBN=0 I guess... which should be allowed?